app/_src/production/dp-config/dpp-on-kubernetes.md

```diff
@@ -175,6 +175,54 @@ control capabilities for the sidecar.
 
 ## Lifecycle
 
+{% if_version gte:2.7.x %}
+The lifecycle management has been significantly improved
+with the `SidecarContainers` feature introduced in Kubernetes v1.29.
+
+### Kubernetes sidecar containers
+
+In order to enable the use of this feature with Kuma you should enable the
+`experimental.sidecarContainers`/`KUMA_EXPERIMENTAL_SIDECAR_CONTAINERS` option.
+
+This feature supports adding the injected Kuma container to
+`initContainers` with `restartPolicy: Always`, which marks it as a sidecar
+container. Refer to the
+[Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/)
+to learn more about how they work.
+
+In effect, the following lifecycle subsections are irrelevant when using this feature.
```

Comment:
are they? You still need to slow down the app shutdown to give time to the CP to propagate endpoint change, no?

Reply:
Yeah. And I guess the point is that it's not even a mesh-specific concern. It could happen with `Service` or `Ingress`